pkgs/by-name/un/unix-privesc-check/package.nix

```
@@ -0,0 +1,29 @@
+{ lib
+, stdenvNoCC
+, fetchurl
+}:
+
+stdenvNoCC.mkDerivation (finalAttrs: {
+  pname = "unix-privesc-check";
+  version = "1.4";
+
+  src = fetchurl {
+    url = "http://pentestmonkey.net/tools/unix-privesc-check/${finalAttrs.pname}-${finalAttrs.version}.tar.gz";
```

```suggestion
    url = "https://pentestmonkey.net/tools/unix-privesc-check/unix-privesc-check-${finalAttrs.version}.tar.gz";
```
this should not use finalAttrs.pname as a change in pname would bery likely break the URL

---

Fixed